lib/browser.ts

```diff
@@ -672,17 +673,14 @@ export class ProtractorBrowser extends Webdriver {
 
     if (this.ignoreSynchronization) {
       this.driver.get(destination);
-      return this.driver.controlFlow().execute(() => {
-        return this.plugins_.onPageLoad();
-      });
+      return this.driver.controlFlow().execute(() => this.plugins_.onPageLoad()).then(() => {});
```

Comment:
None of the functions generated by `pluginFunFactory` are ever exposed to protractor users, so I'm not bothered by the fact that the promise resolves to something useless.  Instead of this `.then()` block - which is really only here for type checking - why not just cast this to a `() => void`?